android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -615,6 +625,46 @@ public void disableNativeFramesTracking() {
         }
     }
 
+    public void startProfiling(Promise promise) {
+        final WritableMap result = new WritableNativeMap();
+        try {
+            RNSentryModuleImpl.javaScriptExecutorFactory.startSamplingProfiler();
+        } catch (UnsupportedOperationException e) {
```

Comment:
```suggestion
        } catch (Throwable e) {
```

Just for the sake of safety.